products/fuse/get-started.adoc

```diff
@@ -1,48 +1,116 @@
 :awestruct-layout: product-get-started
+:awestruct-interpolate: true
 
-## JDK Advice
-Make sure you have a supported JDK installed. OpenJDK or Oracle JDK 6 or 7 is required. Java 8 is not yet supported, but will be in a future release.
+== Prerequisites section title
+Prerequisites &#38; Required Dependencies Checklist
 
-## Extra installation instructions
+== Prerequisites section
+1. Java SE Development Kit (JDK), we recommend OpenJDK or Oracle JDK.
+2. JBoss&#174; Developer Studio 8.1
 
-[start=4]
-. Start JBoss Fuse.
+== Setup Duration
+10 minutes
+
+== Install Duration
+5 minutes
+
+== Build Duration
+20 minutes
+
+== Setup Environment
+
+1.  Install the Java SE Development Kit (JDK), we recommend OpenJDK or
+Oracle JDK.
+2.  Download link:http://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-standalone_jar.jar[JBoss Developer Studio 8.1.0.GA] 
+3.  Run JBoss Developer Studio installer +
+For Mac / Windows Development Hosts:
+a.  Go to the folder you have downloaded JBoss Developer Studio:
+...  Right Click on *jboss-devstudio-8.1.0.GA-installer-standalone.jar*
+... Select _"Open With -> Jar Launcher"_
 +
-For Linux or Mac::
-+    
-    cd jboss-fuse-6.1.0.redhat-379
-    bin/fuse
+For Linux Development Hosts:
 +
-For Windows::
+Go to the folder where you have downloaded JBoss Developer Studio, and execute the
+installer +
 +
-    cd jboss-fuse-6.1.0.redhat-379
-    bin\fuse.bat
+`java -jar jboss-devstudio-8.1.0.GA-installer-standalone.jar`
 +
-. The first time JBoss Fuse is started you must configure an `administrator` user. In the Fuse shell execute the following command:::
+*Note:* This will execute the JBoss Developer Studio installation wizard. +
+4.  Start JBoss Developer Studio. +
+For more details, view the https://access.redhat.com/documentation/en-US/Red_Hat_JBoss_Developer_Studio/8.1/html/Install_Red_Hat_JBoss_Developer_Studio/Install_JBoss_Developer_Studio_Stand-alone_and_JBoss_EAP.html[JBoss
+Developer Studio documentation.]
+5.  Configure Maven
+a.  Open JBoss Developer Studio preferences and expand _JBoss Tools_ and
+select _JBoss Maven Integration._
+b.  Click _Configure Maven Repositories…_ button.
+c.  Click _Add Repository_ and select the
+*redhat-techpreview-all-repository* profile and then click Ok. +
 +
-      esb:create-admin-user
+[.content-img]
+image:#{cdn(site.base_url + '/images/products/devstudio/images_products_devstudio_devstudio-overview-1.png')}[Add Maven Repository]
+d.  Click Finish +
 +
-And type in a username and password.
+[.content-img]
+image:#{cdn(site.base_url + '/images/products/devstudio/images_products_devstudio_devstudio-overview-2.png')}[Configure
```

Comment:
This line and the next line are on two lines, separating the title capture (the text between `[` and `]`). This is breaking the build.